### ISSUE TITLE
add a Popolo JSON export of data in the site

### ISSUE THREAD

**Issue by mhl** [mysociety/yournextrepresentative#590]
In addition to the CSV download (which doesn't include alternative names, for example) and the API (which would need multiple calls to extract data in bulk) it would be good to have an export that produces Popolo JSON for the site (augmented with the data from the `*Extra` models).

The `Election` model could be exported as Popolo Events, as @tmtmtmtm suggested (see #428)

**Comment by akuckartz** [comment]
Please use JSON-LD.

**Comment by akuckartz** [comment]
This is a partial duplicate of #364

**Comment by tmtmtmtm** [comment]
@akuckartz Popolo JSON is something that is well used, well supported, and well understood. Popolo JSON-LD currently isn't — and I'm unaware of anyone else requesting it.

If you were to submit a pull request yourself that implements this, I suspect it would be accepted. But in the absence of that, your best approach would be to either explain why this would _actually_ be useful (rather than just theoretically so), or to at least put together a full, detailed, example of what you would expect the output to look like. It would be a lot easier for someone to generate a target output than to figure out what that output should be.

**Comment by akuckartz** [comment]
@tmtmtmtm I will not try to explain here why Linked Data is useful in practice. That has been discussed again and again elsewhere. If I am the only one here suggesting to provide Linked Data then that is how it is. Network effects can be helpful but they also can delay progress.

**Comment by tmtmtmtm** [comment]
@akuckartz I'm aware of the theoretical reasons why it would be useful — that's why I distinguished that. If there were people actively requesting a JSON-LD output so that they could immediately use it in some project where plain JSON was insufficient, then knowing about that would help us weigh up the relative priority compared to the many other things we want to add. But in the absence of that, the best way to bump this up the priority list, other than actually submitting the PR yourself, is to point us at a detailed enough example output for it become a simple matter of implementation, rather than investigating _what_ to do.

**Comment by wfdd** [comment]
One obvious use case is i18n - which should be familiar to you, having had to extend Popolo's `other_name` schema with a `lang` field in everypolitician.

**Comment by tmtmtmtm** [comment]
@wfdd Yes — this is probably the main thing that has _almost_ pushed us to providing a JSON-LD output in EveryPolitician. But so far there's been no-one making a strong enough case for that, or giving us detailed enough pointers on _how_ we should be doing it, to justify spending the time on investigating what exactly would actually be involved. (I believe the same holds true for YNR too.) If we actually _needed_ this for an actual (not just theoretical) use case, we would take the time to investigate it. Or if someone helped with enough of the prep-work, showing us what the target output should look like, so that it's just a (presumably relatively straightforward) implementation issue, we'd happily do this. But in the absence of either, there are just too many other pressing priorities at the moment.

**Comment by wfdd** [comment]
I don't think that it's pressing to provide a JSON-LD export in YNR (a JSON-LD _import_ would be much preferable, but also kinda pointless right now [#677]); even when I've had to provide translations in three different languages, I've not particularly longed for JSON-LD.  But, in the long-ish term, it might be worth considering whether you wanna lead the trend on its adoption.
